.ci/Jenkinsfile

```diff
@@ -281,32 +282,69 @@ pipeline {
       }
     }
     stage('Release') {
-      options { skipDefaultCheckout() }
+      options {
+        skipDefaultCheckout()
+        timeout(time: 12, unit: 'HOURS')
```

Comment:
This might be unneeded. We can remove it or shorten it.

Reply:
This will help to run the release with an input approval and enough time. IMO, If no input approval then it's unnecessary

Reply:
Thanks. Removed.